### PR TITLE
Move ConnectionTarget to its own file

### PR DESCRIPTION
**Pull request by teinarss**

### CODE REVIEW

File: OpenRA.Game/Network/ConnectionTarget.cs [hunk 1]
```diff
@@ -0,0 +1,67 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
```
Comment (abcdefg30) [inline OpenRA.Game/Network/ConnectionTarget.cs]: 2021

Comment (teinarss) [reply OpenRA.Game/Network/ConnectionTarget.cs]: This will haunt me until we update all the copyrights headers... :(

**Comment by teinarss** [comment]
Fixed

**Review by pchote — APPROVED** [review]

**Review by reaperrr — APPROVED** [review]

**Comment by reaperrr** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/0b774c7c7dc9ef74417d337c68a5a0101b10353b...13f89d35b2a38bfeb3a053e207a18fbbc9aedace

**Pull request merged**